backend/apps/accounts/jwt.py

```
@@ -0,0 +1,116 @@
+
+    def verify(self):
+        """Verify token with dynamic salt."""
+        # First perform standard verification
+        super().verify()
+        
+        # Get user from token
+        user_id = self.get("user_id")
+        try:
+            user = User.objects.get(pk=user_id)
+        except User.DoesNotExist:
+            raise ValueError("User not found")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)